### ISSUE TITLE
Composite automatic sort doesn't work on fixed containers

### ISSUE THREAD

**Issue by diegohaz** [reakit/reakit#827]
### Current behavior

When re-ordering composite items, the focus order is not automatically updated.

### Steps to reproduce the bug

Provide a repo or sandbox with the bug and describe the steps to reproduce it.

1. Open sandbox: https://codesandbox.io/s/peaceful-fire-55x13?file=/src/App.js
2. Navigate through the grid with arrow keys
3. Press "move down" or "move up" to change the order

### Expected behavior

The focus order should be automatically updated.
